Tests/Marketplace/zip_packs.py

```diff
@@ -1,20 +1,19 @@
 import argparse
 import json
 import os
-from concurrent.futures import ThreadPoolExecutor
 import shutil
 import sys
 import logging
-from time import sleep
 from zipfile import ZipFile
-from Tests.Marketplace.marketplace_services import init_storage_client
 from Tests.Marketplace.marketplace_constants import IGNORED_FILES, PACKS_FULL_PATH
 from Tests.scripts.utils.log_util import install_logging
 from demisto_sdk.commands.common.tools import LooseVersion, str2bool
+from Tests.Marketplace.marketplace_constants import GCPConfig
+import subprocess
+from pathlib import Path
 
 ARTIFACT_NAME = 'content_marketplace_packs.zip'
 MAX_THREADS = 4
```

Comment:
You can delete this global variable since you've removed the use of threads.
```suggestion
```

Reply:
Fixed.